howso/hierarchy.amlg

```diff
@@ -80,30 +76,28 @@
 
 
 	;Execute any method in the API directly on any child trainee of this trainee, used for hierarchy operations.
-	;
-	;parameters:
-	; method: name of method to execute
-	; payload: payload to pass to the method
-	; child_name_path: optional, list of strings specifying the user-friendly path of the child subtrainee for execution of method
-	; child_id: optional, id of child trainee to execute method. Ignored if child_name_path is specified
-	; as_external: optional, boolean, applicable only to 'load_subtrainee' and 'save_subtrainee' and if specifying child_name_path or child_id.
-	;	For 'save_subtrainee', stores the child out as an independent trainee and removes it as a contained entity.
-	;	For 'load_subtrainee' updates hierarchy by adding the child as an independently stored trainee to the hierarchy without loading the trainee as a subtrainee.
-	; load_external_trainee_id: optional, trainee id of trainee being loaded, must be specified only when 'load_subtrainee' and as_external is true.
 	#execute_on_subtrainee
 	(declare
 		(assoc
 			;{type "assoc"}
+			;payload to pass to the method
 			payload (assoc)
 			;{type "string"}
+			;name of method to execute
 			method ""
 			;{type "list" values "string"}
+			;optional, list of strings specifying the user-friendly path of the child subtrainee for execution of method
 			child_name_path (null)
 			;{type "string"}
+			;optional, id of child trainee to execute method. Ignored if child_name_path is specified
 			child_id (null)
 			;{type "boolean"}
+			;optional, boolean, applicable only to 'load_subtrainee' and 'save_subtrainee' and if specifying child_name_path or child_id.
+	;	For 'save_subtrainee', stores the child out as an independent trainee and removes it as a contained entity.
+	;	For 'load_subtrainee' updates hierarchy by adding the child as an independently stored trainee to the hierarchy without loading the trainee as a subtrainee.
```

Comment:
Align indentation here please

Reply:
Address in d8a9404f922319cfd8e1c60555d514603ff45ec3